### PR TITLE
[Plugin] Add timeout of connection when downloading the plugins from URL

### LINKED ISSUE #3754: [Plugin] install plugin block when downloading from URL
**Describe the bug**
Install plugin process blocks when downloading from URL.

**To Reproduce**
install a plugin from an invalid URL which has some network problem.

**Expected behavior**
Throw a timeout exception

### PR DESCRIPTION
**Pull request by morningman**
Fix #3754

If no timeout is set, the download process may be blocked forever.

### CODE REVIEW

**Review by yangzhg — DISMISSED** [review]
LGTM

File: fe/src/main/java/org/apache/doris/plugin/PluginZip.java [hunk 1]
```diff
@@ -113,13 +113,13 @@ Path downloadRemoteZip(Path targetPath) throws IOException, UserException {
         cleanPathList.add(zip);
 
         // download zip
-        try (InputStream in = openUrlInputStream(source)) {
+        try (InputStream in = Util.getInputStreamFromUrl(source, null, 5000, 5000)) {
```
Comment (kangkaisen) [inline fe/src/main/java/org/apache/doris/plugin/PluginZip.java:116]: Should be static const value.
If the timeout isn't a config, I think set a larger value maybe better when the plugin resource is big and network is poor.

Comment (morningman) [reply fe/src/main/java/org/apache/doris/plugin/PluginZip.java:116]: I change it to a config

**Review by kangkaisen — DISMISSED** [review]
+1, LGTM

**Review by EmmyMiao87 — APPROVED** [review]

**Pull request merged**